logging/google/cloud/logging/logger.py

```diff
@@ -379,13 +406,17 @@ def log_text(self, text, labels=None, insert_id=None, severity=None,
 
         :type timestamp: :class:`datetime.datetime`
         :param timestamp: (optional) timestamp of event being logged.
+
+        :type resource: :class:`~google.cloud.logging.resource.Resource`
+        :param resource: (Optional) Monitored resource of the entry
```

Comment:
Ditto here on the "defaults to the..." language. It's **extra** important since we want users to be clear on the fact that the batch's resource will not be used by default.